ts/src/okx.ts

```diff
@@ -157,24 +157,26 @@ export default class okx extends Exchange {
                         'market/tickers': 1,
```

Comment:
@sc0Vu this diff makes it very hard to understand what changed here, can't we keep the currency endpoints in the same place?